### PR TITLE
Add imagePullSecrets to cerebro

### PR DESCRIPTION
**Pull request by vipulagarwal**
Signed-off-by: Vipul Agarwal <va@vagarwal.net>

#### What this PR does / why we need it:
`imagePullSecrets` is required when using a private container repository. This PR will handle `imagePullSecrets` if specified in the values.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/cerebro/Chart.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ sources:
  - https://github.com/lmenezes/cerebro-docker
  - https://github.com/lmenezes/cerebro
 name: cerebro
-version: 0.5.2
+version: 0.5.3
```
Comment (cpanato) [inline stable/cerebro/Chart.yaml]: let's update the version to 1.0.0 since this chart is in stable folder :)
```suggestion
version: 1.0.0
```

Comment (vipulagarwal) [reply stable/cerebro/Chart.yaml]: @cpanato: I used the github suggestion commit feature to accept your changes and it has created a DCO problem. Shall I try to revert the commit or would you be able to add your signature?

**Comment by cpanato** [comment]
@vipulagarwal please sign the DCO

**Comment by vipulagarwal** [comment]
> @vipulagarwal please sign the DCO

@cpanato I did sign it but the bot thinks you need to sign this commit:https://github.com/helm/charts/pull/11418/commits/ff7998f6ec7ba2840daaa48880cc574796bc8e0e

```Commit sha: ff7998f, Author: Carlos Tadeu Panato Junior, Committer: GitHub; Expected "Carlos Tadeu Panato Junior ctadeu@gmail.com", but got "Vipul Agarwal va@vagarwal.net".```

I used the github suggestion commit feature to accept your changes and it has created a DCO problem. I will close this PR and create a fresh one. Sorry about that.

**Pull request closed without merging**